faq.md

```diff
@@ -90,3 +90,9 @@ First you must [disconnect](disconnecting-nosto-from-store-front.md) the existin
 
 After you have disconnected an account you can re-connect the account by following the [normal Oauth flow](getting-started.md#connecting-with-an-existing-nosto-account)
 
+## Placeholder image for products without images
+
+Placeholder image will be used for the products without images. 
```

Comment:
Let's rephrase it like:

In case a product does not have any image, the Nosto module will use the configured placeholder as the image for the product.